### PR TITLE
newer alpine (for newer PHP) and apk upgrade #1655

### PR DESCRIPTION
**Pull request by dougbreaux**
Apologize, I don't have the setup to run the unit tests and not worth that level of effort to me for this trivial of a change. Thus, feel free to ignore this PR. But I had already done the change and run it in local Docker, so I thought I'd put the PR out there just in case.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -46,7 +46,8 @@ RUN apk --update --no-cache add \
         php7-xml \
         php7-simplexml \
         php7-zlib \
-        s6
+        s6 \
+	&& apk -U upgrade
```
Comment (ArthurHoaro) [inline Dockerfile:50]: What's the point of running an upgrade? 
If I'm not mistaken, existing packages are based on Alpine tag, and installed ones should grab the latest version available.

Comment (dougbreaux) [reply Dockerfile:50]: So I could be mistaken on this, and maybe Alpine is somewhat different than some other distros, but what I understood from a container security best-practices talk is that the Linux distro version Docker image just reflects the last time a full major release was made, but that individual packages within the distro aren't guaranteed to be at their latest levels. So this is to catch those up. I'll see if I can find a reference.

Comment (dougbreaux) [reply Dockerfile:50]: Quick search looks like there are mixed opinions on whether the potential security gains are worth the potential breakages. So, either way, IMO.

Comment (nodiscc) [reply Dockerfile:50]: I'm not in favor of upgrading the packages in the image. This makes the build process grab different package versions depending on when the build was performed, leading to unreproducible builds/problems. I'd rather watch the Alpine [releases](https://wiki.alpinelinux.org/wiki/Alpine_Linux:Releases) feed and perform manual base image upgrades via Pull Requests

**Review by ArthurHoaro — APPROVED** [review]

**Comment by ArthurHoaro** [comment]
Thanks!

**Pull request merged**